src/app/courses/progress-courses/courses-progress-chart.component.html

```diff
@@ -13,7 +13,7 @@
   </ng-container>
 </div>
 <div class="errors errors-index" #errorsIndex>
-  <div class="anchor-label" i18n>Steps</div>
+  <div class="anchor-label" i18n>{{label}}</div>
```

Comment:
Instead of directly showing the label variable in the template, you should change this to use the i18n select syntax.  This will allow the text to be translated.

https://angular.io/guide/i18n#select-among-alternative-text-messages

Reply:
I think I got this right. I wasn't sure if I needed to change anything in the i18n folder so I just left it alone.